src/Microsoft.Spatial/Build.NuGet/Microsoft.Spatial.Release.nuspec

```diff
@@ -11,7 +11,7 @@
     <iconUrl>http://static.tumblr.com/hgchgxz/9ualgdf98/icon.png</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>Contains classes and methods that facilitate geography and geometry spatial operations. Supports OData v4 only.</summary>

```

Comment:
>Supports OData v4 only [](start = 101, length = 22)

please remove "Support OData v4 only".  This library sits below OData and is not OData specific.